app/components/forms/order-form.js

```diff
@@ -0,0 +1,150 @@
+import Ember from 'ember';
+import FormMixin from 'open-event-frontend/mixins/form';
+import { countries } from 'open-event-frontend/utils/dictionary/demography';
+
+const { Component, computed } = Ember;
+
+export default Component.extend(FormMixin, {
+  holders: [1, 2, 3],
+  fillDetails() {
```

Comment:
Sure. Will help. 

- First, remove this method. 
- Have an object called as `buyer` to have the details of the buyer.
- Bind that to the buyer data input fields
- Then replace `holders: [1, 2, 3]` with a proper array that has holder objects that could store the firstname, lastname, email and checked status. Like 

	```js
	holders: [
		{
			firstName: '',
			lastName: '',
			email: '',
			sameAsBuyer: false
		},
		{
			firstName: '',
			lastName: '',
			email: '',
			sameAsBuyer: false
		}
	]
	```
- Now, loop through this to display the holder forms and also bind the data to each of the input.
- Use [`ui-checkbox`](https://semantic-org.github.io/Semantic-UI-Ember/#/modules/checkbox) component

	```hbs
	{{ui-checkbox label=(t 'Same as Ticket Buyer') checked=holder.sameAsBuyer onChange=(action 'fillHolderData' holder)}}
	```
- Now, define an action called as `fillHolderData` which would get two arguments. The checkbox value and the that specific holder object. You can use get to get the data from buyer object and set to set the data into the holder object.
- Done. 

> Please not the code snippets are not just for example purposes for you to understand what I mean. Just copy pasting them directly may or may not for.